src/lib/Form/DataTransformer/VersionInfoTransformer.php

```diff
@@ -50,19 +39,13 @@ public function transform($value)
             );
         }
 
-        return ['content_info' => $value->getContentInfo(), 'version_no' => $value->versionNo];
+        return [
+            'content_info' => $value->getContentInfo(),
+            'version_no' => $value->versionNo,
+        ];
     }
 
-    /**
-     * @param array|null $value
-     *
-     * @return VersionInfo|null
-     *
-     * @throws TransformationFailedException
-     * @throws UnauthorizedException
-     * @throws NotFoundException
```

Comment:
Those two are still valid, require FQCN though.

_Side: seems to me that they should be caught actually and re-thrown as `TransformationFailedException`_.

Reply:
> Side: seems to me that they should be caught actually and re-thrown as TransformationFailedException.

I agree.

Reply:
Fixed in https://github.com/ezsystems/ezplatform-admin-ui/pull/1148/commits/42437d627f79cccd4225b508580a7512e00f0d8d